types/stripe/index.d.ts

```diff
@@ -37,87 +37,93 @@
 /// <reference types="node" />
 
 declare class Stripe {
-    DEFAULT_HOST: string;
```

Comment:
This file uses 4 space indents - can you avoid making unrelated changes here?

Reply:
Ups, sorry about that